hack/tree_status.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
could we rename this file to hack/verify-go-modules.sh?

Reply:
Yes, changed it.